### DISCUSSION TITLE
All recognized and untagged faces should be visible in People Section

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente/ente#4091] [Enhancements]
while reviewing face suggestions under people, it shows some other close faces. but those are not created as a new face. would be good if all scanned faces are shown under people without names, for us to tag with names later. recognized as faces but not tagged are not visible under people section, but available under faces in each individual photos. this makes it difficult to navigate to each photo to tag new faces manually.

so all faces recognized should be listed in People as Untagged, so this can be created as new face or merged with existing faces.

**Comment by koerli89** [comment]
I agree it's highly needed. Especially since the face recognition algorithm is not very accurate (yet?). I want to see the faces the algorithm detects but cannot fully attribute it to one existing person.

Also instead of having to scroll through all faces in the face selection menu, it would be helpful to have the **most likely listed first**

**Comment by EcksDy** [comment]
I've also noticed that although all of my images are indexed and synced, some faces will be detected and then unrecognized only when I manually open the details of the image.
This is somewhat weird, I'd love to be sure that all of the photos were scanned and any faces it's unsure about appear near the people section as suggested here.

**Comment by pyxl-dev** [comment]
I totally agree for this feature ! It's pretty annoying to click on "Details" just to make sure it detects and attribute all people on the media.

**Comment by Mape6** [comment]
I want to add here that assigned faces to peoples by the AI should be overwriteable.
The AI assigned peoples to some picture which are wrong I can't find a way to assigned the pictures to the right peoples.

**Comment by smunk303** [comment]
I agree. This feature is a must!

**Comment by Bsaubolle** [comment]
I'd like to emphasize this feature, because right now the manual process is very laborious without a specific sort. Having a dedicated feed or sort by unnamed would make it tremendously easier on the user to begin grouping faces.

**Comment by koerli89** [comment]
I'd like to add to my previous comment. It should be possible to reattribute faces to another person where there was a mismatch.

**Comment by burdiyan** [comment]
This is currently a deal breaker for me to move to Ente. Manually going to the details of each photo to categorize the person is very annoying. I wish it would just aggregate all the recognized but unassigned faces somewhere.

**Comment by brog-io** [comment]
This has been added in the latest release

**Comment by laurenspriem** [comment]
Closing this as it has been added in the latest release.

**Reply by addisonklinke** [reply]
@laurenspriem at least on the desktop UI v1.7.22, I saw there were a lot of unrecognized faces after the initial time I enabled machine learning. However, after categorizing a few of them (with many more remaining), the other suggestions are no longer showing. I know they were largely different people, so it's not just a matter of being grouped into the newly named ones. Is there anything I can do if the "People" tab isn't showing the unnamed faces anymore?

**Reply by AswinAsok** [reply]
nice find @addisonklinke 

the AllPeople dialog was added quite recently and it seems i missed this feature to show all clusters.
I have added it in #10223 and this will be available in the v1.7.23 release going out next week.

**Comment by burdiyan** [comment]
This is still a problem in the iOS app I think. There's no easy way to see all the photos with untagged faces. There's no way to make a smart album like that either. You have to fiddle with lots of menus and basically go photo by photo to make sure every photo has all the faces tagged properly.

This is super annoying, and it's the only thing that stops me from using Ente really. I just want to have a single place with all the photos with faces untagged, such that I could dismiss them, or tag all the faces properly.
